Composer/packages/server/src/models/utilities/util.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export const getDialogFileName = (file: string) => {
+  if (file.endsWith('.dialog')) {
```

Comment:
nitpick

I think u can simplify this `file.match(/(.+?)(\.[^.]*$|$)/).`This way u can get the extension and the name of the file in 1 line rather than endsWith and another substring function

Reply:
The dialog file is a.dialog, while qna file is a.en-us.qna. The locale cannot be removed. So I change it in another way.